scality_manila_utils/helper.py

```diff
@@ -0,0 +1,146 @@
+# Copyright (c) 2015 Scality
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import io
+import os
+import os.path
+import signal
+
+from scality_manila_utils import utils
+from scality_manila_utils.export import ExportTable
+from scality_manila_utils.exceptions import (EnvironmentException,
+                                             ExportException)
+
+
+class Helper(object):
```

Comment:
I'm a bit dubious on the use of a class for this, and making is stateful (`self.exports`) for CLI usage. Any benefit over plain procedures in a module which read/update/write the exports file (or whatever actions they ought to take) with all required parameters passed as arguments?